### ISSUE TITLE
Some ideas for next year

### ISSUE THREAD

**Issue by jzaefferer** [jquery/gsoc#6]
- Tell students to contact us about project ideas before starting a proposal. We can recommend unpopular projects, which can massively increase the odds of getting accepted.
- Tell students (even more?) to get involved early on, maybe after submitting the initial proposal. Contributing code, and more than just a few lines, makes a big difference in getting accepted. A good contributor can get accepted even with a rather bad proposal.
- Tell promising students that applied for a project with tough competition to consider apply for something else (see also first point above)
- tbc

**Comment by dylans** [comment]
I would add:
- Start thinking of project ideas after this summer, once we see what
  worked and what was a struggle
- Get people interested in mentoring earlier
- Blog about the SoC process so people know it's happening
- Tell Google to fix their terribly slow SoC application review app. ;)

Regards,
-Dylan

on 4/23/16, 03:30 (GMT-07:00) Jörn Zaefferer said the following:

**Comment by manrajgrover** [comment]
@jzaefferer I would suggest having some rewarding student projects where they pick up a project and work on it for a long time and actually own that project. They can be mentored accordingly. At the end of project, jQuery Foundation can award them with some goodies and certification. I believe its one good way to get new contributors involved. Thoughts?

**Comment by jzaefferer** [comment]
Thanks @dylans, those are good points. 

@ManrajGrover thanks for the suggestion, but I feel like those are rather out of scope. Specifically:

> I would suggest having some rewarding student projects where they pick up a project and work on it for a long time and actually own that project

For us the point of the program is to get students involved in our existing projects and community. Creating new projects from scratch would mean they likely get very little contact with the existing community, and I doubt that every student is interesting in owning a project.

> At the end of project, jQuery Foundation can award them with some goodies and certification.

Isn't the money paid out by the program a decent extrinsic motivation, along with all the intrinsic motivation to participate (experience, fame, whatnot)? I'm not sure that some goodies would make a difference.

**Comment by manrajgrover** [comment]
@jzaefferer Sorry, I'll make my point more clear.

> For us the point of the program is to get students involved in our existing projects and community.
> Creating new projects from scratch would mean they likely get very little contact with the existing
> community, and I doubt that every student is interesting in owning a project.

What I meant was, some projects that were in this year's [jQuery's ideas list](https://github.com/jquery/gsoc/wiki/GSoC-2016-Ideas-List) but were not selected in current program could be given out to interested students and mentored. This will engage more students to participate. At the end of project(with stipulated deadline), performance can be evaluated.

Regarding certification, what I wanted to suggest is apart from GSoC, more students can get involved through these student-projects. Yes, GSoC is a great platform. But not every student gets to reap benefits of it and this will keep them engaged.

**Comment by jzaefferer** [comment]
>  Yes, GSoC is a great platform. But not every student gets to reap benefits of it and this will keep them engaged.

GSoC is a great platform for attracting students to participating projects. I don't see how we could replace that platform given our resources. And given that we only used 9 out of the 10 slots we, because we couldn't find a 10th student that seemed like a good match, I don't think our own program has that much potential.

**Comment by kumarmj** [comment]
Hi, please let me know whether jQuery is coming this year in GSOC. I  am a little contributer in QUnit & jQuery, I would love to do more awesome work, I would love to get suggestion about projects too.

**Comment by jzaefferer** [comment]
No, it isn't. Would be great to have you contribute to our projects anyway.
